src/components/Header/index.tsx

```diff
@@ -86,17 +82,32 @@ const Header = ({
         <div className="nav__endwrapper">
           <ul className="right-container">
             <li className="nav__tabs nav__tabs--right">
-              <button
-                type="button"
-                className="dark-mode-toggle"
-                onKeyPress={keyPressDarkModeHandler}
-                onMouseDown={darkModeController.handleEvent}
-                onMouseUp={darkModeController.handleEvent}
-              >
-                <span className="sr-only">Toggle Dark Mode</span>
-                <i className="material-icons light-mode-only">nights_stay</i>
-                <i className="material-icons dark-mode-only">wb_sunny</i>
-              </button>
+              <ThemeToggler>
```

Comment:
I left this in here instead of extracting to another component because its the only one using the `ThemeToggler`. I could be convinced to extract this out though